superset/charts/commands/data.py

```diff
@@ -67,8 +67,8 @@ def run(self, **kwargs: Any) -> Dict[str, Any]:
 
         return return_value
 
-    def run_async(self) -> Dict[str, Any]:
-        job_metadata = async_query_manager.init_job(self._async_channel_id)
+    def run_async(self, user_id: int) -> Dict[str, Any]:
```

Comment:
`user_id` will be `None` for anonymous (public) users: https://github.com/maxcountryman/flask-login/blob/89ce0167a8603d2c53fa2d0d604a20bfe03c0eaf/flask_login/mixins.py#L59